### PR TITLE
web front end updates for python3, add the example

### PR DESCRIPTION
**Pull request by wallacbe**
This change is [Reviewable](https://reviewable.io/reviews/google/openhtf/729)

### CODE REVIEW

**Comment by wallacbe** [comment]
PTAL

File: openhtf/output/servers/station_server.py [hunk 1]
```diff
@@ -42,7 +41,6 @@
 
 STATION_SERVER_TYPE = 'station'
 
-MIMETYPE_REVERSE_MAP = {v: k for k, v in mfg_inspector.MIMETYPE_MAP.iteritems()}
```
Comment (Kenadia) [inline openhtf/output/servers/station_server.py]: Seems like this is still needed

**Comment by wallacbe** [comment]
File "/home//venv/local/lib/python2.7/site-packages/openhtf-1.2.2-py2.7.egg/openhtf/output/servers/station_server.py", line 23, in <module>
    from openhtf.output.callbacks import mfg_inspector
  File "/home//venv/local/lib/python2.7/site-packages/openhtf-1.2.2-py2.7.egg/openhtf/output/callbacks/mfg_inspector.py", line 48, in <module>
    from openhtf.output.proto import guzzle_pb2
  File "/home//venv/local/lib/python2.7/site-packages/openhtf-1.2.2-py2.7.egg/openhtf/output/proto/guzzle_pb2.py", line 23, in <module>
    serialized_pb=_b('\n!openhtf/output/proto/guzzle.proto\x12\x07openhtf\"N\n\x0fTestRunEnvelope\x12*\n\x0cpayload_type\x18\x02 \x01(\x0e\x32\x14.openhtf.PayloadType\x12\x0f\n\x07payload\x18\x03 \x01(\x0c*S\n\x0bPayloadType\x12\x17\n\x13\x43OMPRESSED_TEST_RUN\x10\x01\x12\x13\n\x0f\x43OMPRESSED_VARZ\x10\x02\x12\x16\n\x12\x43OMPRESSED_LOGFILE\x10\x03')
TypeError: __init__() got an unexpected keyword argument 'syntax'

**Comment by wallacbe** [comment]
Works if I manually remove all of the   syntax='proto2' in the generated py files, strange

**Comment by wallacbe** [comment]
protoc --version
libprotoc 3.0.0

if it matters

**Pull request merged**